sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/LogsIngestionAsyncClient.java

```diff
@@ -38,7 +41,20 @@
 import static com.azure.monitor.ingestion.implementation.Utils.gzipRequest;
 
 /**
- * The asynchronous client for uploading logs to Azure Monitor.
+ * This class provides an asynchronous client for uploading custom logs to Azure Monitor's Log Analytics workspace.
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p>To create an instance of the {@link LogsIngestionClient}, use the {@link LogsIngestionClientBuilder} and configure
+ * the various options provided by the builder to customize the client as per your requirements. There are two required
+ * properties that should be set to build a client:
+ * <ol>
+ * <li>{@code endpoint} - The <a href="https://learn.microsoft.com/azure/azure-monitor/essentials/data-collection-endpoint-overview?tabs=portal#create-a-data-collection-endpoint">data collection endpoint</a>.
+ * See {@link LogsIngestionClientBuilder#endpoint(String) endpoint} method for more details.</li>
+ * <li>{@code credential} - The AAD authentication credential that has the "Monitoring Metrics Publisher" role assigned to it.
```

Comment:
Let's use the new name for AAD, which is Entra ID. I have a Germanium issue assigned to myself to track a bulk update across all repos.

Reply:
Discussed with @scottaddie offline. We'll keep this as AAD until the Azure Identity docs are updated.